src/identity/hooks/useVerified.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+import { base } from 'viem/chains';
+import type { Address } from 'viem';
+
+import { getEASAttestations } from '../getEASAttestations';
+import { useIdentity } from './useIdentity';
+
+export function useVerified(address: Address) {
+  const [verified, setVerified] = useState(false);
+  const identity = useIdentity();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const attestation = await getEASAttestations(address, base);
+      const found = attestation.find(({ schemaId }) => schemaId === identity.schemaId);
+
+      if (found !== undefined) {
+        setVerified(true);
+      }
+    };
+
+    fetchData();
+  }, [address]);
+
+  return verified;
```

Comment:
so `useVerified` return if an address is verified, and the schema comes from the `identity.schemaId`.

But in the same app, I can see we use multiple schemas. So I think we should be more explicit in the params of the hook what schema we are checking.

Reply:
`useVerified` as name feels a bit too vague